tests/renderer/xml/test_view_service.py

```diff
@@ -26,23 +26,38 @@ def test_empty():
 
 def test_reference_wms():
     map = ViewServiceRecord(
-        reference_wms='http://example.com',
+        reference_wms='http://example.com?SERVICE=WMS&REQUEST=GetMap&FORMAT=image/png&SRS=epsg:2056',
```

Comment:
example.com is a registered  domain. 
Wouldn't it be better to use an URI to a server that does not exist, or is under our control? You never know what the owner of this domain decides to send back as an error message besides the fact, that it is probably pretty anoying for them to get all this fake requests and mails... unless they analyze them for marketing purposes. 😉

Reply:
ok ok, I agree... Even when we never call it in the end. But I will change it to a better one.

Reply:
thanks - my bad! not awake enough 🙈

Reply:
That not necessary. As written on http://example.com/:

> **Example Domain**
>
> This domain is established to be used for illustrative examples in documents. You may use this domain in examples without prior coordination or asking for permission.

https://www.iana.org/domains/reserved

Reply:
Puhh saved a lot of work :wink: